repos/known-good.proj

```diff
@@ -20,7 +20,7 @@
     <RepositoryReference Include="xliff-tasks" />
 
     <!--RepositoryReference Include="msbuild" /-->
-    <!--RepositoryReference Include="clicommandlineparser" /-->
+    <RepositoryReference Include="clicommandlineparser" />
```

Comment:
This should be uncommented in cli.proj too--I think your branch forked off before @dseefeld commented out the cli.proj refs in https://github.com/dotnet/source-build/pull/463.

Reply:
Thanks, fixed.